core/src/main/java/jenkins/util/VirtualFile.java

```diff
@@ -66,7 +66,6 @@
 import org.apache.tools.ant.types.selectors.TokenizedPath;
 import org.apache.tools.ant.types.selectors.TokenizedPattern;
 import org.kohsuke.accmod.Restricted;
-import org.kohsuke.accmod.restrictions.Beta;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
```

Comment:
BTW @Wadeck there were some APIs you introduced later with a review comment saying that they should be unrestricted in a weekly; did you forget?

Reply:
Actually they are not yet ready / finalized